==================================== INPUT ====================================
# expected-to-throw

fragment refetchableWithoutNameConflictWithRefetchableF1 on Node
  @refetchable
{
  __typename
}

fragment refetchableWithoutNameConflictWithRefetchableF2 on Node
  @refetchable(queryName: "refetchableWithoutNameConflictWithRefetchableF1RefetchQuery")
{
  id
}
==================================== ERROR ====================================
✖︎ Duplicate definition for @refetchable operation 'refetchableWithoutNameConflictWithRefetchableF1RefetchQuery' from fragments 'refetchableWithoutNameConflictWithRefetchableF1' and 'refetchableWithoutNameConflictWithRefetchableF2'

  refetchable_without_name_conflict_with_refetchable.invalid.graphql:3:10
    2 │ 
    3 │ fragment refetchableWithoutNameConflictWithRefetchableF1 on Node
      │          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    4 │   @refetchable

  ℹ︎ also defined here

  refetchable_without_name_conflict_with_refetchable.invalid.graphql:9:10
    8 │ 
    9 │ fragment refetchableWithoutNameConflictWithRefetchableF2 on Node
      │          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   10 │   @refetchable(queryName: "refetchableWithoutNameConflictWithRefetchableF1RefetchQuery")
